### PR TITLE
DDP-7559 . Update cloud-secretmanager version

### PR DESCRIPTION
**Pull request by ssettipalli**
look at https://broadinstitute.atlassian.net/browse/DDP-7559
Latest version of cloud-secretmanager should have worked but not clear why its running into issue in DSS CustomExportJob
Downgrading to old repo version (1.5.1 as in dss-core) seem to work. 
Need to revisit and upgrade to latest some point of time.

### CODE REVIEW

File: pepper-apis/parent-pom.xml [hunk 1]
```diff
@@ -487,10 +487,6 @@
             <artifactId>fst</artifactId>
             <version>2.57</version>
         </dependency>
-        <dependency>
```
Comment (ssettipalli) [inline pepper-apis/parent-pom.xml:490]: duplicate and hence removed

File: pepper-apis/parent-pom.xml [hunk 2]
```diff
@@ -515,6 +511,7 @@
         <dependency>
             <groupId>com.google.cloud</groupId>
             <artifactId>google-cloud-secretmanager</artifactId>
+            <version>1.5.1</version>
         </dependency>
 
         <dependency>
```
Comment (MocanaAtBroad) [inline pepper-apis/parent-pom.xml:517]: parent-pom.xml is no longer used.

Comment (ssettipalli) [reply pepper-apis/parent-pom.xml:517]: we will be deleting this parent-pom eventually. So will leave it for now

File: pepper-apis/dsm-core/pom.xml [hunk 3]
```diff
@@ -246,7 +246,7 @@
         <dependency>
             <groupId>com.google.cloud</groupId>
             <artifactId>google-cloud-secretmanager</artifactId>
-            <version>2.0.5</version>
+            <version>1.5.1</version>
```
Comment (MocanaAtBroad) [inline pepper-apis/dsm-core/pom.xml]:
```suggestion
          
```

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**